new/LIT6981PPSorcerers.xml

```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6981PPSorcerers" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against sorcerers</title>
+                <title xml:lang="gez" xml:id="t2">ጸሎት፡ ወስእለት፡ በእንተ፡ እለ፡ ይቀትሉ፡ ሰብአ፡ ወመሠርያን፡</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Ṣalot wa-sǝʾlat baʾǝnta ʾǝlla yǝqattǝlu sabʾa wa-maśarrǝyān…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Magic"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2024-05-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <note>The following incipit and explicit are taken from <ref type="mss" corresp="BAVet128"/>.</note>
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <ab>
+                     በስመ፡ አብ፡ <gap reason="ellipsis"/>ጸሎት፡ ወስእለት፡ በእንተ፡ እለ፡ ይቀትሉ፡ ሰብአ፡ ወመሠርያን፡ <sic>ወ</sic>ዝውእቱ፡ ወዘያስተኃፍሮሙ፡ ላዕለ፡ መሠርያን፡ አስማተ፡ እግዚአብሔር።
```

Comment:
Why did you highlight the ወ before ዝውእቱ, but not the ወ before ዘያስተኃፍሮሙ. The second seems to me more odd than the first one.

Reply:
I followed the catalogue's annotations